res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,444 @@
+  };
+}
+
+Jockey3ME.effectParamLedSet = function (currentDeck, index, status, control) {
+  var ledValue = engine.getParameter("[EffectRack1_EffectUnit" + currentDeck + "_Effect1]", "parameter" + index);
+  ledValue = parseInt(ledValue * 127);
+  midi.sendShortMsg(status,control,ledValue);
+}
+
+Jockey3ME.effectMixLedSet = function (currentDeck, status, control) {
+  var ledValue = engine.getParameter("[EffectRack1_EffectUnit" + currentDeck + "]", "mix");
+  ledValue = parseInt(ledValue * 127);
+  midi.sendShortMsg(status,control,ledValue);
+}
+
+Jockey3ME.effectSelectLedSetNumEffect = function (currentDeck, status, control, value) {
+  // var ledValue = engine.getValue("[EffectRack1_EffectUnit" + currentDeck + "]", "num_effects");
+  var ledValue = Jockey3ME.effectsAvailable; // num_effects returns how many effects in Unit is, not what is available. Set to 5
+  if ((Jockey3ME.num_effectsValue[currentDeck - 1] + value) > ledValue) {
+    Jockey3ME.num_effectsValue[currentDeck - 1] = 1;
+  } else {
+    Jockey3ME.num_effectsValue[currentDeck - 1] += value;
+  }
+  var newLedValue = parseInt((Jockey3ME.num_effectsValue[currentDeck - 1] / ledValue) * 127);
+  midi.sendShortMsg((status-32),control,newLedValue);
+}
+
+Jockey3ME.effectSelect = function (channel, control, value, status, group) {
+  var currentDeck = parseInt(group.substring(23,24));
+  engine.setValue("[EffectRack1_EffectUnit" + currentDeck + "]", "chain_selector", (value-64));
+
+  // Set Leds
+  Jockey3ME.effectSelectTimer = engine.beginTimer(100, "Jockey3ME.effectSelectLedSet(" + status + "," + currentDeck + ")",1);
+  Jockey3ME.effectSelectLedSetNumEffect(currentDeck,status,92,(value-64));
+}
+
+Jockey3ME.effectSelectLedSet = function (status, currentDeck) {
+  status -= 32;
+  var num_parameters = engine.getValue("[EffectRack1_EffectUnit" + currentDeck + "_Effect1]", "num_parameters");
+  if (num_parameters > 3) {num_parameters = 3;};
+  if (num_parameters) {
+    for (var i = 1, j = 30; i <= num_parameters; i++) {
+      Jockey3ME.effectParamLedSet(currentDeck, i, status, j);
+      j++;
+    };
+  }
+  Jockey3ME.effectMixLedSet(currentDeck, status, 29);
+  // Jockey3ME.effectSelectLedSetNumEffect(currentDeck,status,92);
+}
+
+// Browser Knop to Browse the Playlist
+Jockey3ME.TraxEncoderTurn = function (channel, control, value, status, group) {
+    var newValue = (value-64);
+
+   engine.setValue(group,"SelectTrackKnob",newValue);
+
+}
+
+// Browser Knop with Shift to Browse the Playlist Tree
+Jockey3ME.ShiftTraxEncoderTurn = function (channel, control, value, status, group) {
+    var newValue = (value-64);
+
+   if (newValue == 1) engine.setValue(group,"SelectNextPlaylist",newValue);
+   else engine.setValue(group,"SelectPrevPlaylist",1);
+}
+
+Jockey3ME.loop_double_halve = function (channel, control, value, status, group) {
+  var newValue = (value-64);
+
+  if (newValue == 1) {
+    engine.setValue(group,"loop_double",1);
+    engine.setValue(group,"loop_double",0);
```

Comment:
Like with hotcue_X_clear, there is no need to set loop_double or loop_halve to 0